cmd/promql-langserver/promql-langserver.go

```diff
@@ -37,12 +52,27 @@ func main() {
 	}
 	if config.RESTAPIPort != 0 {
 		fmt.Fprintln(os.Stderr, "REST API: Listening on port ", config.RESTAPIPort)
+
+		r := prometheus.NewRegistry()
+		r.MustRegister(httpRequestsTotal)
+		r.MustRegister(httpRequestDuration)
```

Comment:
we should pass the registry as an argument to the `CreateHandler` func and register the metrics in there, otherwise we are mixing the concerns of this main package a bit too much.

Reply:
okay